form-ics213.html

```diff
@@ -14,6 +14,14 @@
   <body>
     <div class="version">0.1.0</div>
     <div id="loading"><div>Loading<div id="spin"><div id="spin_1" class="spin"></div></div></div></div>
+    <div id="err">
```

Comment:
As usual with boilerplate, I suggest moving this chunk into resources/html/xxx.html and using a data-include-html to insert it here and in the other forms.

Reply:
I completely agree that minimizing the boilerplate in the main form file is an important objective. In this case however, since we want to be able to present errors regardless of whether the include files were included successfully or not, it needs to be in the main file boilerplate.